core/src/main/java/cz/xtf/core/openshift/OpenShift.java

```diff
@@ -74,6 +74,8 @@
 
 	public static final String KEEP_LABEL = "xtf.cz/keep";
 
+	public static final int DEFAULT_TIMEOUT = 120_000;
```

Comment:
If we are going to make it public, we should probably have a more descriptive name and some javadoc for it.  Perhaps DEFAULT_HTTP_TIMEOUT ?  This value is basically used for a mix of various timeouts related to okhttp (connect, read and write timeouts) and the only reason they are being set is binary s2i builds.